specs/src/specs/params.md

```diff
@@ -42,9 +42,9 @@ are blocked by the `x/paramfilter` module.
 | ibc.Transfer.SendEnabled | true | Enable sending tokens via IBC. | True |
 | ibc.Transfer.ReceiveEnabled | true | Enable receiving tokens via IBC. | True |
 | slashing.SignedBlocksWindow | 5000 | The range of blocks used to count for downtime. | True |
-| slashing.MinSignedPerWindow | 5 | Minumum signatures in the block. | True |
+| slashing.MinSignedPerWindow | 0.05 (5%) | Minimum proportion of blocks signed per window to avoid being slashed. | True |
 | slashing.DowntimeJailDuration | 10 mins | Duration of time a validator must stay jailed. | True |
-| slashing.SlashFractionDoubleSign | 5.0% | Percentage slashed after a validator is jailed for downtime. | True |
+| slashing.SlashFractionDoubleSign | 5.0% | Percentage slashed after a validator is jailed for double-signing. | True |
```

Comment:
Could we keep the format for percentages constant i.e. if we're using decimal with parentheses for `MinSignedPerWindow` can we do the same for `SlashFractionDoubleSign`

Reply:
This table isn't consistent w/ respect to the format you described. For example:

```
| slashing.SlashFractionDowntime | 1.0% |
```

and

```
| staking.MinCommissionRate | 0.05 (5%) |
```

I can make it consistent but noting that this will bloat the diff in this PR.

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/2479

Reply:
@cmwaters is your feedback blocking or can I tackle it in a different PR?